controllers/providers/assistant/gslb.go

```diff
@@ -290,8 +321,11 @@ func (r *Gslb) GetExternalTargets(host string, extClusterNsNames map[string]stri
 		clusterTargets := getARecords(a)
 		if len(clusterTargets) > 0 {
 			targets = append(targets, clusterTargets...)
-			log.Info().Msgf("Added external %s Gslb targets from %s cluster", clusterTargets, cluster)
+			log.Info().
+				Str("cluster targets", fmt.Sprintf("%s", clusterTargets)).
+				Str("cluster", cluster).
+				Msg("Extend Gslb targets by targets from cluster")
 		}
 	}
-	return
+	return targets
```

Comment:
nothing against(I actually like this style more) but why are we returning explicitly now?

Reply:
Hey, good point, mee too, BUT:

```go
golangci-lint run
controllers/providers/assistant/gslb.go:330:2: naked return in func `GetExternalTargets` with 38 lines of code (nakedret)
        return
        ^
make: *** [lint] Error 1
```

The linter basically says,  the function is too large to have naked return (nakedret static analysis).

	
The function was extended by splitting the log messages into more lines 

```go
		log.Warn().
			Str("fqdn", fqdn).
			Str("nameserver", nameserver).
			Err(err).
			Msg("Can't resolve FQDN using nameserver")
```

I had three options at the end:  satisfy nakedret, refactor the function into two funcs, or leave the logs on one line? 

Finally I decided to satisfy nakedret.

Reply:
Very clear, thanks!